tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "target": "ES2022",
```

Comment:
I feel this conservative target makes sense, only downside is that it may make the JS bundle a bit larger than ESNext